reporter/src/main/kotlin/reporters/freemarker/FreemarkerTemplateProcessor.kt

```diff
@@ -335,6 +336,23 @@ class FreemarkerTemplateProcessor(
         @Suppress("UNUSED") // This function is used in the templates.
         fun filterForUnresolvedVulnerabilities(vulnerabilities: List<Vulnerability>): List<Vulnerability> =
                 vulnerabilities.filterNot { input.resolutionProvider.isResolved(it) }
+
+        /**
+         * Return a flag indicating that issues have been encountered during the run of an advisor with the given
+         * [capability][capabilityStr] with at least the given [severity][severityStr]. This typically means that
+         * the report is incomplete; therefore, it should contain a corresponding warning.
+         */
+        @JvmOverloads
+        fun hasAdvisorIssues(capabilityStr: String, severityStr: String = Severity.ERROR.name): Boolean {
```

Comment:
Is it really necessary that this function takes the string representations instead of the typed values? IMO we should rather make the `enum` classes available to the template.

Reply:
Maybe this helps:
* https://freemarker.apache.org/docs/pgui_misc_beanwrapper.html#jdk_15_enums
* https://stackoverflow.com/a/44723453/1127485

Reply:
The BeansWrapper mechanism is now used to make the enums available to the template.